python/orca/src/bigdl/orca/learn/pytorch/utils.py

```diff
@@ -184,7 +184,7 @@ def stats(self, mean=True, last=False):
 
 
 def find_free_port():
-    with closing(socket.socket(socket.AF_INET, socket.SOCK_STREAM)) as s:
+    with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
```

Comment:
Will this impact? @leonardozcm

Reply:
`with` will close s as soon as the following action is done, so it matters not I suppose.